pyznap/utils.py

```diff
@@ -100,9 +102,12 @@ def read_config(path):
             else:
                 if option in ['key']:
                     dic[option] = value if os.path.isfile(value) else None
-                elif option in ['frequent', 'hourly', 'daily', 'weekly', 'monthly', 'yearly']:
+                elif option in [
+                    'frequent', 'hourly', 'daily', 'weekly', 'monthly', 'yearly',
+                    'retry', 'retry_interval', 'ServerAliveInterval', 'ServerAliveCountMax'
```

Comment:
From my comment in the `main.py` file: If these are set as per-dest values, then this will not work anymore, we will have to treat them similar to the `raw_send` option.